lib/stdlib/src/parse_groks.rs

```diff
@@ -265,6 +303,31 @@ mod test {
             tdef: TypeDef::object(Collection::any()).fallible(),
         }
 
+        presence_of_alias_sources_argument {
+            args: func_args![
+                value: r##"2020-10-02T23:22:12.223222Z info 200 hello world"##,
+                patterns: Value::Array(vec![
+                    "%{common_prefix} %{_status} %{_message}".into(),
+                    "%{common_prefix} %{_message}".into(),
+                    ]),
+                aliases: value!({
+                    "common_prefix": "%{_timestamp} %{_loglevel}",
+                    "_timestamp": "%{TIMESTAMP_ISO8601:timestamp}",
+                    "_loglevel": "%{LOGLEVEL:level}",
+                    "_status": "%{POSINT:status}",
+                    "_message": "%{GREEDYDATA:message}"
+                }),
+                alias_sources: Value::Array(vec![]),
```

Comment:
I would like to see a test with a least 1 real alias file loaded. If it's easier, you can use the more general VRL testing framework at `lib/tests/tests/functions/...` and include a file containing an alias.